invoicing_app/tests/tests.py

```diff
@@ -169,12 +174,57 @@ def test_localize_date_old(self):
         my_date = dt(2020, 4, 11, 0, 0)
         self.assertEqual(
             str(self.my_command.localize_date('AR', my_date)),
-            "2020-04-11 00:00:00-03:54"
+            '2020-04-11 00:00:00-03:54'
+        )
+        self.assertEqual(
+            str(self.my_command_new.localize_date('AR', my_date)),
+            '2020-04-11 00:00:00-03:54'
+        )
+
+    def test_today_date_incorrect_format(self):
+        # Not a string in correct format date
+        self.options['today_date'] = 'a.asd-asd??++*asd'
+        with self.assertRaises(Exception):
+            self.my_command.handle(**self.options)
+        with self.assertRaises(Exception):
+            self.my_command_new.handle(**self.options)
+
+    # Only for old script
```

Comment:
I think that it is better to have separated tests (and very likely in different test suites) for the new script and the old one, that way you shouldn't need a comment to clarify if a test if for the old script or not. For example, I see that the name of this test suite is `TestScriptGenerateTaxReceiptsOld` but has tests for the old and new script.

Reply:
Sorry, I didn't refresh the page, these comments seem to be similar to what Alex pointed.

Reply:
Thanks! I'll do it.